types/knockout/index.d.ts

```diff
@@ -1,13 +1,14 @@
-// Type definitions for Knockout v3.4.0
+﻿// Type definitions for Knockout v3.4.0
 // Project: http://knockoutjs.com
 // Definitions by: Boris Yankov <https://github.com/borisyankov>, 
 //                 Igor Oleinikov <https://github.com/Igorbek>, 
 //                 Clément Bourgeois <https://github.com/moonpyk>, 
 //                 Matt Brooks <https://github.com/EnableSoftware>, 
 //                 Benjamin Eckardt <https://github.com/BenjaminEckardt>, 
 //                 Mathias Lorenzen <https://github.com/ffMathy>
+//                 Ken Kohler <https://github.com/kkohler2>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// TypeScript Version: 2.3
+// TypeScript Version: 2.6.3
```

Comment:
Any reason for a version bump here? This PR doesn't seem to require bumping version as far as I can tell.

Reply:
Yes.  I was using TypeScript 2.6.3 when I ran into the compiler errors.  Using an older version of TypeScript, possibly 2.2 or 2.3, I am not seeing the errors.  Updating the version is for matching a TypeScript version I can confirm is working with the change.